src/io.c

```diff
@@ -837,3 +837,39 @@ add_filter_module(nmsgtool_ctx *c, const char *args) {
 
 	my_free(tmp);
 }
+
+void
+add_stats_module(nmsgtool_ctx *c, const char *args) {
```

Comment:
Please document what's in args

Reply:
This was mostly a cut-and-paste of the filter module `add_filter_module` function above, which is similarly (un)documented.

While I'm normally on board with more commentary, I don't think a comment to the effect of:
    `/* args contains a module name and parameter separated by a comma. */`
would add value to the short bit of code a few lines down:
```
	/* Parse arguments */
	tmp = my_strdup(args);
	mod_name = strtok_r(tmp, ",", &saveptr);
	mod_param = strtok_r(NULL, "", &saveptr);
```
particularly given that this is not a public API, and its audience is developers modifying `nmsgtool` itself, for whom the above code snippet will be in the same page of code with any reasonably-placed comment to its effect.

From a _user_ of the code perspective, this will be documented in the command line help and the man page.